delfin/drivers/utils/ssh_client.py

```diff
@@ -278,3 +280,44 @@ def do_exec(self, command_str):
                 'is not a recognized command' in result:
             raise exception.StorageBackendException(result)
         return result
+
+    def do_exec_command(self, command_list):
```

Comment:
Function name might be named `do_exec_shell` or `do_exec_channel`, or other names.

Reply:
has been modify